.clang-tidy

```
@@ -0,0 +1,5 @@
+Checks: ' clang-diagnostic-*, clang-analyzer-*, clang-analyzer-core.*, clang-analyzer-cplusplus.*, clang-analyzer-nullability.*, clang-analyzer-unix.*, performance-*, bugprone-use-after-move, bugprone-move-forwarding-reference, cppcoreguidelines-rvalue-reference-param-not-moved, misc-unused-parameters '
```

Just a nitpick, you can express the checks as an array, which I think is more readable when there are so many.
```suggestion
Checks:
  - clang-diagnostic-*
  - clang-analyzer-*
  - clang-analyzer-core.*
  - clang-analyzer-cplusplus.*
  - clang-analyzer-nullability.*
  - clang-analyzer-unix.*
  - performance-*
  - bugprone-use-after-move
  - bugprone-move-forwarding-reference
  - cppcoreguidelines-rvalue-reference-param-not-moved
  - misc-unused-parameters
```

---

Thank you!